<launch>
  <node pkg="motor_control" type="motor_control_node.py" name="motor_control_node" output="screen">
    <rosparam command="load" file="$(find motor_control)/config/parameters.yaml" />
  </node>
</launch>
